### DISCUSSION TITLE
重启丢失配置文件

### DISCUSSION THREAD

**Discussion by GuanWX** [AlistGo/alist#1352] [Q&A]
## 重启丢失配置文件
我是使用docker进行部署，重启之后会丢失之前配置的全部信息，包括账号信息，密码等内容。如何修改配置文件使其重启也能生效？

**Comment by Becods** [comment]
缺少持久化
按照 [文档](https://alist-doc.nn.ci/en/docs/install/docker) 里提供的命令运行

**Reply by 544325999** [reply]
按照命令启动但是还是缺少持久化
docker run -d --restart=always -v /etc/alist:/opt/alist/data -p 5244:5244 --name="alist" xhofe/alist:latest

**Reply by ghost** [reply]
> 按照命令启动但是还是缺少持久化 docker run -d --restart=always -v /etc/alist：/opt/alist/data -p 5244：5244 --name=“alist” xhofe/alist：latest

把-v /etc/alist:opt/alist/data 改成-v D:/alist:/opt/alist/data，也就是 -v  【宿主机文件路径】：【docker-alist文件路径】
